src/utils/string.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+/**
+ * Takes a string and returns the string with public URLs removed.
+ * It doesn't remove the URLs like `chrome://..` because they are internal URLs
+ * and they shouldn't be removed.
+ */
+export function removeURLs(
+  string: string,
+  removeExtensions: boolean = true
+): string {
+  const regExpExtension = removeExtensions ? '|moz-extension' : '';
+  const regExp = new RegExp(
+    '((https?|ftp' + regExpExtension + ')://)([^\\s/$.?#].[^\\s)]*)',
```

Comment:
I think this regex could have a bit of inline documentation like this:

```
'((https?|ftp' + regExpExtension + ')://)([^\\s/$.?#].[^\\s)]*)',
// ^                                       ^           ^
// |                                       |           Document what this matches
// |                                       Document what this matches
// Document what this matches
```

It doesn't have to be much, but some words here would be appreciated.